doc/releases/changes.rst

```diff
@@ -8,6 +8,8 @@ Version 5.3.0 (unreleased month day, year)
 
 Enhancements
 ~~~~~~~~~~~~
+- Add "jupyter" renderer which uses JupyterChart for rendering (#3283)
```

Comment:
```suggestion
- Add "jupyter" renderer which uses JupyterChart for rendering (#3283). See :ref:`renderers` for more information.
```
nit: Maybe adding this link is helpful so that users can read up on it?